src/lib.rs

```diff
@@ -254,10 +311,14 @@ impl<T: Sync + Send + 'static> Nucleo<T> {
         }
     }
 
-    pub fn update_config(&mut self, config: MatcherConfig) {
+    pub fn update_config(&mut self, config: Config) {
         self.worker.lock().update_config(config)
     }
 
+    /// The main way to interact with the matcher, this should be called
+    /// regulrarly (for example each time a frame is rendered). To avoid
+    /// excessive redraws this method will wait `timeout` miliiseconds for the
+    /// worker therad to finish. Its recommend to set the timeout to 10ms.
```

Comment:
```suggestion
    /// regularly (for example each time a frame is rendered). To avoid
    /// excessive redraws this method will wait `timeout` milliseconds for the
    /// worker therad to finish. It is recommend to set the timeout to 10ms.
```